src/instr/instr_paje_values.cpp

```diff
@@ -31,9 +31,9 @@ value::value(const char* name, const char* color, type_t father)
 
 value::~value()
 {
-  xbt_free(name);
+/*   xbt_free(name);
```

Comment:
This should not be commented. These variables were created in the constructor and should thus be freed in the destructor.